From 420c1a73416a607cf9157520e86c8b21e778576d Mon Sep 17 00:00:00 2001
From: Starbuck5 <46412508+Starbuck5@users.noreply.github.com>
Date: Sat, 19 Nov 2022 21:17:32 -0800
Subject: [PATCH] Adjust surface masks to create a proper 24 bit Surface

The SDL 2.26.0 pre-release rejects a 24 bit Surface created with this mask, which is the right call, as it needs 32 bits of space. Older SDL went ahead and created a 32 bit surface anyway, which it probably shouldn't have.
---
 test/surface_test.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/surface_test.py b/test/surface_test.py
index 1c2369b366..51e2bc822f 100644
--- a/test/surface_test.py
+++ b/test/surface_test.py
@@ -2023,7 +2023,7 @@ def test_get_losses(self):
             mask8 = (224, 28, 3, 0)
             mask15 = (31744, 992, 31, 0)
             mask16 = (63488, 2016, 31, 0)
-            mask24 = (4278190080, 16711680, 65280, 0)
+            mask24 = (16711680, 65280, 255, 0)
             mask32 = (4278190080, 16711680, 65280, 255)
 
             # Surfaces with standard depths and masks
